### PR TITLE
REL: 0.16.0

### PR DESCRIPTION
**Pull request by skoudoro**
# Summary

Preparation for 0.16.0 release, targeting Friday, March 8.  The last 0.X.X series. DIPY 1.0.0 in June!!!

It will be good to merge the following PR before the release on Friday.

- [x] #1748 (@skoudoro)
- [x] #1743 (@Garyfallidis )
- [x] #1714 (@jhlegarreta ) 
- [ ] #1230 (@RafaelNH )
- [x] #1726 (@ShreyasFadnavis)
- your suggestion ?

Just a note that any of this PR can block the release. If they are not in, it will be for the next release in
june.

# Release checklist
- [x] add release_notes 0.16.0
- [x] Update changelog an api_changes
- [x] Update .mailmap
- [x] Set release number in info.py and doc/conf.py
- [x] Check the copyright years in doc/conf.py and LICENSE
- [x] Check documentation
- [x] Check Tutorial
- [x] Update index.rst
- [x] Update Highligth.rst, old_news.rst, stateofheart.rst
- [x] Update developers.rst list

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Just rebased #1714 on master. The previously reported CI build failures are clearly unrelated to the PR, so to me the PR is ready to be merged.

**Comment by skoudoro** [comment]
@Garyfallidis, @arokem, Can you have a quick look on release summary?

File: Changelog [hunk 1]
```diff
@@ -24,6 +24,18 @@ Dipy
 
 The code found in Dipy was created by the people found in the AUTHOR file.
 
+* 0.16.0 (Sunday, 10 March 2019)
+
+- Horizon, fast, modular, unified and adaptive visualization
+- New Tractometry tools : Bundle Analysis / Bundle Profiles
+- New reconstruction model: IVIM Mix
+- New command line interface: Affine and Diffeomorphic Registration
+- New command line interface: Probalistic, Deterministic and PFT Tracking
```
Comment (Garyfallidis) [inline Changelog]: Probabilistic

File: Changelog [hunk 2]
```diff
@@ -24,6 +24,18 @@ Dipy
 
 The code found in Dipy was created by the people found in the AUTHOR file.
 
+* 0.16.0 (Sunday, 10 March 2019)
+
+- Horizon, fast, modular, unified and adaptive visualization
```
Comment (Garyfallidis) [inline Changelog]: Horizon, medical visualization interface powered by QuickBundlesX

File: Changelog [hunk 3]
```diff
@@ -24,6 +24,18 @@ Dipy
...
+- New Tractometry tools : Bundle Analysis / Bundle Profiles
+- New reconstruction model: IVIM Mix
```
Comment (Garyfallidis) [inline Changelog]: New reconstruction model : IVIM MIX.

File: doc/index.rst [hunk 4]
```diff
@@ -13,18 +13,17 @@ visualization, and statistical analysis of MRI data.
...
-- New continuous integration with AppVeyor CI.
-- Nibabel Streamlines API now used almost everywhere for better memory management.
-- Compatibility with Python 3.7.
-- Many tutorials added or updated (5 New).
-- Large documentation update.
-- Moved visualization module to a new library: FURY.
-- Closed 287 issues and merged 93 pull requests. 
+**DIPY 0.16.0** is now available. New features include:
+
+- Horizon, fast, modular, unified and adaptive visualization
```
Comment (Garyfallidis) [inline doc/index.rst]: Update as above

File: doc/api_changes.rst [hunk 5]
```diff
@@ -5,6 +5,23 @@ API changes
 Here we provide information about functions or classes that have been removed,
 renamed or are deprecated (not recommended) during different release circles.
 
+DIPY 0.16 Changes
+-----------------
+
+**Stats**
+
+Welcome to the new module ``dipy.viz.stats``. This module will be use to integrate various analysis.
```
Comment (jhlegarreta) [inline doc/api_changes.rst]: _(...)will be use to integrate various analysis._ -> _(...) will be used to integrate various analyses._

**Comment by RafaelNH** [comment]
@skoudoro ! I've just noticed that my PR #1230 is pushing this back! I think, I've addressed all the comments made on that PR! I will just rebase and update the PR. Give me some minutes!

**Comment by skoudoro** [comment]
Ok @RafaelNH , I give you an hour. But I need to release tonight.

**Comment by RafaelNH** [comment]
@skoudoro! PR #1230 completed! Let's just see if all checks pass!

**Comment by RafaelNH** [comment]
@skoudoro I think we still need to wait a bit before the checks passing! If you cannot wait longer, don't worry, you can proceed with the new release! I've wrote you just because I thought you were waiting for this PR. It would be cool to have this new reconstruction model already in release 0.16 . But, if it is not the case, it can also be the first PR being merged to the master branch for release 1.0 ;).

**Pull request merged**